src/shared/requester.js

```diff
@@ -242,6 +250,7 @@ export const getRepositories = async ({
   organizationName,
   numberOfProjects,
 }) => {
+  console.log('Im fetching repos !');
```

Comment:
Delete console.log from this part of code

Reply:
Oops, fixed